ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/objectstore/jdbc/JDBCImple_driver.java

```diff
@@ -102,13 +102,13 @@ public boolean commit_state(Uid objUid, String typeName)
 
 			int rowcount2 = pstmt2.executeUpdate();
 			if (rowcount2 > 0) {
+                connection.commit();
```

Comment:
Use of spaces instead of tabs (ie not conforming to the existing formatting of the file)